py_mind_memo/view.py

```
@@ -638,22 +668,67 @@ def on_insert_icon(self, event):
+        top.title("操作マニュアル - py_mind_memo")
+        top.transient(self.root)  # メインウィンドウに紐付け
+
+        # クロスプラットフォーム対応の最大化
+        if sys.platform == "win32":
+            top.state("zoomed")
+        elif sys.platform == "darwin":
+            # macOSはzoomed相当なし; 画面サイズに合わせてウィンドウを設定
+            top.geometry(f"{top.winfo_screenwidth()}x{top.winfo_screenheight()}+0+0")
+        else:
+            # Linux (X11)
+            top.attributes("-zoomed", True)
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)